optashift-employee-rostering-gwtui/src/main/super/org/optaplanner/openshift/employeerostering/gwtui/emulate/java/time/LocalDateTime.java

```diff
@@ -480,6 +480,10 @@ public long getLong(DateTimeField field) {
 
     if (field instanceof ChronoField) {
       ChronoField f = (ChronoField) field;
+      if (f.equals(ChronoField.INSTANT_SECONDS)) {
+          return this.time.getSecond() + this.time.getMinute() * 60 + this.time.getHour() * 60 * 60 +
+                  this.date.toEpochDay() * 60 * 60 * 24;
+      }
```

Comment:
And these, where do they come from?

Reply:
The ability to directly add a `Duration` to `LocalDateTime`; mostly for my sanity